config/routes.rb

```diff
@@ -21,10 +21,10 @@
     post '/sign_up/register' => 'sign_up/registrations#create', as: :sign_up_register
     get '/sign_up/start' => 'sign_up/registrations#show', as: :sign_up_start
     get '/sign_up/verify_email' => 'sign_up/emails#show', as: :sign_up_verify_email
-    get '/sign_up/create_password' => 'sign_up/confirmations#show', as: :user_confirmation
-    get '/sign_up/enter_email_again' => 'sign_up/confirmations#new', as: :new_user_confirmation
-    post '/sign_up/register_again' => 'sign_up/confirmations#create', as: :sign_up_register_again
+    get '/sign_up/enter_password' => 'sign_up/passwords#new', as: :sign_up_new_password
```

Comment:
If these controllers no longer use devise, does that mean we can move them outside the devise block in our router?